translations/it.yml

```diff
@@ -83,19 +83,19 @@ create:
 error:
     poll:
         decryptionFailed:
-            title: 'Decryption failed'
-            description: 'Decrypting poll data failed. This is most likely caused by a wrong encryption key. Please double-check the URL entered, especially the part after the question mark.'
+            title: 'La decrittazione e fallita.'
+            description: 'La decrittazione è fallita. Il più probbabile sia dovuto ad una chiave cirttografica sbagliata. Per favor, controlla la URL che hai inserito, particolarmente la parte dopo il punto interrogativo.'
```

Comment:
La decifratura è fallita. Questo errore è probabilmente dovuto ad una chiave crittografica non corretta. Controlla la URL che hai inserito, ed in particolare la parte che segue il punto interrogativo.

Reply:
```suggestion
            description: 'La decifratura è fallita. Questo errore è probabilmente dovuto ad una chiave crittografica non corretta. Controlla la URL che hai inserito, ed in particolare la parte che segue il punto interrogativo.'
```